modules/administration/pages/troubleshooting/tshoot-hardware-refresh.adoc

```diff
@@ -0,0 +1,27 @@
+[[troubleshooting-hardware-refresh]]
+= Troubleshooting hardware refresh
```

Comment:
@keichwa I don't think we need this at all. Only the note that you have added in [modules/reference/pages/systems/system-details/sd-hardware.adoc](https://github.com/uyuni-project/uyuni-docs/pull/2492/files#diff-4acdfbeacb8e95bfd93183ab07616efbd73af95409ad6c330afb07ec7132ee77) should suffice to close this issue.

Reply:
ok, yes.  thank you for checking!